main.nf

```diff
@@ -250,10 +250,8 @@ workflow NEXTFLOW_WGS {
 	ch_output_info = ch_output_info.mix(bamtoyaml.out.bamchoice_INFO)
 
 	ch_bam_start_dedup_dummy = Channel.empty()
-	if(params.run_melt) {
-		dedupdummy(ch_bam_start)
-		ch_bam_start_dedup_dummy = dedupdummy.out.dedup_dummy
-	}
+	dedupdummy(ch_bam_start)
```

Comment:
Always produce dedupdummy when starting from bam. This allows the QC postprocessing to run also in non-melt cases.